shared/ios/KeybaseShare/ShareViewController.m

```diff
@@ -25,64 +25,6 @@ @interface ShareViewController ()
 
 @implementation ShareViewController
 
-- (NSItemProvider*)firstSatisfiesTypeIdentifierCond:(NSArray*)attachments cond:(BOOL (^)(NSItemProvider*))cond {
-  for (NSItemProvider* a in attachments) {
-    if (cond(a)) {
-      return a;
-    }
-  }
-  return nil;
-}
-
-- (NSMutableArray*)allSatisfiesTypeIdentifierCond:(NSArray*)attachments cond:(BOOL (^)(NSItemProvider*))cond {
-  NSMutableArray* res = [NSMutableArray array];
-  for (NSItemProvider* a in attachments) {
-    if (cond(a)) {
-      [res addObject:a];
-    }
-  }
-  return res;
-}
-
-- (BOOL)isWebURL:(NSItemProvider*)item {
-  // "file URLs" also have type "url", but we want to treat them as files, not text.
-  return (BOOL)([item hasItemConformingToTypeIdentifier:@"public.url"] && ![item hasItemConformingToTypeIdentifier:@"public.file-url"]);
-}
-
-// getSendableAttachments will get a list of messages we want to send from the share attempt. The flow is as follows:
```

Comment:
we need to handle this kind of thing after / while we're loading and not way ahead of time